systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
```

Comment:
You should change the name per Ana's suggestion to clarify what these static files are that you are bringing in.